### PR TITLE
LOG-5565 Migrate loki cluster claims to obs-logging

### PR DESCRIPTION
**Pull request by cahartma**
Update cluster_claims stanza to use new obs-logging pools.  
https://issues.redhat.com/browse/LOG-5565

cc @periklis @xperimental @jcantrill 

Any other logging-storage repos I am missing?    
https://docs.google.com/spreadsheets/d/1Xl2hoxFFVyJWi1qohxcQbK2KQEiy4v-6NB1-W5OeyLI/edit#gid=0

### CODE REVIEW

**Comment by periklis** [comment]
/pj-rehearse pull-ci-openshift-loki-release-5.8-test-operator

File: ci-operator/config/openshift/loki/OWNERS [hunk 1]
```diff
@@ -9,6 +9,7 @@ approvers:
 - jcantrill
 - periklis
 - xperimental
+- cahartma
```
Comment (periklis) [inline ci-operator/config/openshift/loki/OWNERS:12]: You are not allowed to change this file, this is sync'ed from the original repo

Comment (cahartma) [reply ci-operator/config/openshift/loki/OWNERS]: reverted

**Pull request merged**